extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/test/MSQTestControllerContext.java

```diff
@@ -353,13 +354,19 @@ public WorkerManager newWorkerManager(
       WorkerFailureListener workerFailureListener
   )
   {
+    MSQWorkerTaskLauncherConfig taskLauncherConfig = new MSQWorkerTaskLauncherConfig();
+    taskLauncherConfig.highFrequenceCheckMillis = 0;
```

Comment:
`0` means busy loop, which seems excessive; how about `1`?

Reply:
setting it to 1 adds `1s` for ~500 cases; it would be nice if it would execute a `sleep(0)` instead of skipping the sleep entirely

set it to `1` for now